src/auth/permissions/__tests__/index.test.integration.ts

```diff
@@ -26,12 +26,18 @@ describe('Permisions Module', () => {
   describe('::grantNew', () => {
     test('SUCCESS - Grant a role a permission entry that doesn\'t already exist', async () => {
       try {
+        const currentPermissions = await knex('permission')
+          .select('')
+          .count()
+          .then((res) => Number(res[0].count));
```

Comment:
Up to you which you go with, just throwing out the equivalent destructuring:
```ts
const [{ count }] = await knex('permission').select().count();
```
Thought that means you have to do the type casting elsewhere. 🤷🏼‍♂️

Reply:
Lol, I initially wrote it like this, but thought it was less readable 🤷🏽‍♀️